app/controllers/renewal/payments_controller.rb

```diff
@@ -36,35 +36,7 @@ def skip
     end
 
     def callback
-      order_id = params[:orderId]
-      result = checkout.fetch_order(order_id: order_id)
-      session[:attempts] ||= 0
-
-      if result.success?
-        amount = result.value
-        Membership.create_for_member(@member, amount: amount, start_membership: true, square_transaction_id: order_id, source: "square")
-        MemberMailer.with(member: @member, amount: amount.cents).renewal_message.deliver_later
-
-        session[:amount] = amount.cents
-
-        redirect_to renewal_confirmation_url, status: :see_other
-        return
-      end
-
-      errors = result.error
-
-      if errors.first[:code] == "NOT_FOUND"
-        # Give Square a little while for the transaction data to be available
-        session[:attempts] += 1
-        if session[:attempts] <= 10
-          render :wait, layout: nil
-          return
-        end
-      end
-
-      Rails.logger.error(errors)
-      session.delete(:attempts)
-      flash[:error] = "There was an error processing your payment. Please come into the library to complete signup."
```

Comment:
This logic has all moved to `SquarePaymentJob`- we just render the confirmation page here and don't attempt to process the payment further.